charts/opensearch/templates/service.yaml

```diff
@@ -32,6 +32,9 @@ spec:
   - name: {{ .Values.service.transportPortName | default "transport" }}
     protocol: TCP
     port: {{ .Values.transportPort }}
+  - name: {{ .Values.service.metricsPortName | default "metrics" }}
+    protocol: TCP
```

Comment:
Do we need this for non-headless service ?
@VILJkid

Reply:
Thanks @prudhvigodithi for the review!
We already have metrics exposed from the headless service, however to make things stable, with ease in monitoring, I added this. Plus, it matches the formatting with OpenSearch Dashboards `Service` template, with explicitly specifying `protocol`.

Let me know your thoughts on this, @prudhvigodithi @peterzhuamazon, so I can make necessary changes.

Reply:
Thanks